### PR TITLE
Fetch data from server to display html on about page

### PR DESCRIPTION
**Pull request by saraholiviawu**
Write JS code that fetches data from server and renders html in the about me page after clicking a button. Change fonts using Google Fonts throughout web pages.

### CODE REVIEW

**Comment by saraholiviawu** [comment]
See small button below image

Screenshot 2020-06-02 at 11 11 15 AM - Display 2

File: portfolio/src/main/webapp/gallery.html [hunk 1]
```diff
@@ -3,14 +3,17 @@
   <head>
     <meta charset="UTF-8">
     <title>Sarah Wu</title>
+    <link href="https://fonts.googleapis.com/css2?family=Yrsa:wght@300;400;500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant:wght@300;500;600;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
-    <link href="https://fonts.googleapis.com/css?family=Shadows+Into+Light|Source+Sans+Pros">
+    <!-- font-family: 'Cormorant', serif; -->
+    <!-- font-family: 'Yrsa', serif; -->
```
Comment (sabadas) [inline portfolio/src/main/webapp/gallery.html:10]: these commented lines can be removed?

File: portfolio/src/main/webapp/gallery.html [hunk 2]
```diff
@@ -3,14 +3,17 @@
...
     <script src="script.js"></script>
   </head>
   <body>
     <div class="container">
+
```
Comment (sabadas) [inline portfolio/src/main/webapp/gallery.html:15]: more for my understanding, but why do you sometimes have blank lines in between but not others? (Just looking at PR where sometimes you added blank line and other times you removed)

File: portfolio/src/main/webapp/projects.html [hunk 3]
```diff
@@ -3,14 +3,21 @@
...
     <meta charset="UTF-8">
     <title>Sarah Wu</title>
+    <link href="https://fonts.googleapis.com/css2?family=Crimson+Text:wght@400;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@100;500;600;700&family=Yrsa:wght@300;400;500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Yrsa:wght@300;400;500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Cormorant:wght@300;500;600;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
-    <link href="https://fonts.googleapis.com/css?family=Shadows+Into+Light|Source+Sans+Pros">
     <script src="script.js"></script>
+    <!-- font-family: 'Crimson Text', serif; -->
```
Comment (sabadas) [inline portfolio/src/main/webapp/projects.html:12]: can be removed?

**Review by peter14f — APPROVED** [review]

File: portfolio/src/main/webapp/gallery.html (hunk 1 above)
Comment (saraholiviawu) [reply portfolio/src/main/webapp/gallery.html:10]: Fixed

File: portfolio/src/main/webapp/projects.html (hunk 3 above)
Comment (saraholiviawu) [reply portfolio/src/main/webapp/projects.html:12]: Fixed

File: portfolio/src/main/webapp/gallery.html (hunk 2 above)
Comment (saraholiviawu) [reply portfolio/src/main/webapp/gallery.html:15]: Fixed. The code now only adds a line space between sections (body / header), not div tags.

**Pull request merged**